src/Table/Footer.tsx

```diff
@@ -16,10 +16,9 @@ export interface Props {
   showLastPage?: boolean;
   isLoading?: boolean;
   lengthUnknown?: boolean;
+  visiblePageRangePadding: number;
```

Comment:
Chatted offline and we've decided to change this to `visiblePageRangeSize` where the value is an enum of odd numbers: `1 | 3 | 5 | 7` (even numbers throw the logic off and also result in an unbalanced UX)